docs/UserGuide.md

```diff
@@ -50,158 +70,247 @@ Some example commands you can try:
 
-Examples:
-* `find n/shingeki s/watching t/seinen` displays all the shows in the Trackermon's show list that contain **shingeki** in NAME, **watching** in STATUS, and **seinen** in TAG. [put image here after ui update]
-* `find n/Shutter Island s/watching` displays all the shows in the Trackermon's show list that contains **Shutter Island** in NAME, and **watching** in STATUS. [put image here after ui update]
+---
+
+### Adding a show: `add`
+
+**Description:** Planning to watch a show and want to store its details? Add a show to Trackermon's show list!
+
+**Format:** `add n/NAME s/STATUS [c/COMMENT] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-warning">:bulb: **Tip:**
-Find is case-insensitive, and the order in which the keywords are entered is irrelevant. Partial words **will** be matched as well. e.g., `shing` will match `shingeki`.
+A show can have any number of comments or tags (including 0)
 </div>
 
-<div markdown="span" class="alert alert-danger">:exclamation: **Caution:**
-find `find n/shingeki n/no n/2` does not mean `find n/shingeki no 2` The former will only find show names that match with **2**(as mentioned in [features](#features)) while the latter will find all show names that match **shingeki, no and 2**. This applies across the name and tag parameters.
-</div>
+**Example & Output:** `add n/All of us are dead s/plan-to-watch t/Horror`
+
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Deleting a show: `delete`
 
-Deletes the specified show from Trackermon.
+**Description:** Wanting to remove an unwanted show? Delete the show at the specified index shown in Trackermon's show list!
+
+**Format:** `delete INDEX`
+
+**Example & Output:** `delete 2`
 
-Format: `delete <INDEX>`
-* Deletes the show at the specified `<INDEX>`.
-* The index refers to the index number shown in the displayed show list. (not overall)
-* The index **must be a positive integer** 1,2,3,..
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
-Examples:
-* `list` followed by `delete 2` removes 2nd show in Trackermon.
-* `find ghibli` followed by `delete 1` removes 1st show in results of `find` command.
+<div markdown="block" class="alert alert-danger"> **:exclamation: Caution:** 
+The **index** parameter provided should be a [non-zero unsigned integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) within the allowed range of Java’s [`int`](#glossary) data type. On top of that, the index should be within the bounds of the show list.<br>
+  e.g. If there are 5 shows saved in the show list, then the valid index ranges from 1 to 5.
+
+</div>
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
-### Editing a show: `Edit`
 
-Edit the specified show from Trackermon.
+### Editing a show: `edit`
+
+**Description:** Wanting to modify a show? Edit a show at the specified index shown in Trackermon's show list!
```

Comment:
Same nitpick as the one on adding a show.

Reply:
👍